dataduct/steps/create_load_redshift.py

```diff
@@ -41,7 +40,7 @@ def __init__(self, id, table_definition, input_node=None,
             script_arguments = list()
 
         script_arguments.extend([
-            '--table_definition=%s' % table_exists_script.sql(),
+            '--table_definition=%s' % table.sql().sql(),
```

Comment:
table.sql() returns a sql statement and then the next sql converts it to a string.